src/data/features.ts

```diff
@@ -1,7 +1,7 @@
 import Cloud from "@fluentui/svg-icons/icons/cloud_24_regular.svg?raw";
 import Tag from "@fluentui/svg-icons/icons/tag_24_regular.svg?raw";
 import TabDesktop from "@fluentui/svg-icons/icons/tab_desktop_20_regular.svg?raw";
-import EyeVisible from "@fluentui/svg-icons/icons/eye_show_20_regular.svg?raw";
+import EyeVisible from "@fluentui/svg-icons/icons/eye_20_regular.svg?raw";
```

Comment:
not quite sure why this is showing as a change.... see that it was merged in recently here (to fix CI I believe) https://github.com/files-community/Website/commit/31b9d6f8a8dddf1fa468abf9f7bcd3a67fccae69

Not sure it's a problem but I wanted to point it out.